include/tables/hive/PARTCOLSTATSTable.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file is part of ePipe
+ * Copyright (C) 2020, Logical Clocks AB. All rights reserved
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+
+#ifndef EPIPE_PARTCOLSTATSTABLE_H
+#define EPIPE_PARTCOLSTATSTABLE_H
+
+#include "tables/DBTable.h"
+
+struct PARTCOLSTATSRow {
+  Int64 mPARTID;
+};
+
+class PARTCOLSTATSTable : public DBTable<PARTCOLSTATSRow> {
+
+public:
+  PARTCOLSTATSTable() : DBTable("PART_COL_STATS") {
+    addColumn("PART_ID");
+  }
+
+  PARTCOLSTATSRow getRow(NdbRecAttr *value[]) {
+    PARTCOLSTATSRow row;
+    row.mPARTID = value[0]->int64_value();
+    return row;
+  }
+
+  void remove(Ndb *conn, Int64 PARTID)
+  {
+    AnyMap key;
+    key[0] = PARTID;
+    int count = deletesByIndex(conn, "PartIndex", key, boost::none);
```

Comment:
Can we add a partition key on PARTID to that table?

Reply:
same as above